virttest/utils_net.py

```diff
@@ -1720,18 +1723,21 @@ def refresh_neigh_table(interface_name=None, neigh_address="ff02::1"):
     for interface in interfaces:
         refresh_cmd = "ping6 -c 2 -I %s %s > /dev/null" % (interface,
                                                            neigh_address)
-        process.system(refresh_cmd, ignore_status=True)
+        func(refresh_cmd)
 
 
-def get_neighbours_info(neigh_address="", interface_name=None):
+def get_neighbours_info(neigh_address="", interface_name=None, session=None):
     """
     Get the neighbours infomation
     """
-    refresh_neigh_table(interface_name, neigh_address)
+    refresh_neigh_table(interface_name, neigh_address, session=session)
+    func = process.getoutput
+    if session:
+        func = session.cmd_output
     cmd = "ip -6 neigh show nud reachable"
     if neigh_address:
         cmd += " %s" % neigh_address
-    output = decode_to_text(process.system_output(cmd))
+    output = func(cmd)
```

Comment:
Seems this way you loose `decode()`, and also `process.system_output` only returns `cmd_result.stdout`,  but `process.getoutput` returns `cmd_result.exit_status, cmd_result.stdout_text`, possibly L#1745 - L#1752 wont work properly.

Reply:
@vivianQizhu I couldn't get you, please correct me if I am wrong,
`process.getoutput()` returns `cmd_result.stdout_text` and property `stdout_text` takes care of decode part as well which returns `stdout`. how does this affect lines that is mentioned ?

Reply:
@balamuruhans You are right. Sorry that I didn't notice the difference between `stdout_text` and `stdout`, and also missed the `[1]` in `getoutput()` return line.